src/coreComponents/mesh/utilities/ComputationalGeometry.hpp

```diff
@@ -446,47 +450,15 @@ void getBoundingBox( localIndex const elemIndex,
  */
 GEOSX_HOST_DEVICE
 inline
-real64 hexVolume( real64 const X[][3] )
+real64 hexVolume( real64 const (&X)[8][3] )
 {
-  real64 X7_X1[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X[7] );
-  LvArray::tensorOps::subtract< 3 >( X7_X1, X[1] );
-
-  real64 X6_X0[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X[6] );
-  LvArray::tensorOps::subtract< 3 >( X6_X0, X[0] );
-
-  real64 X7_X2[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X[7] );
-  LvArray::tensorOps::subtract< 3 >( X7_X2, X[2] );
-
-  real64 X3_X0[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X[3] );
-  LvArray::tensorOps::subtract< 3 >( X3_X0, X[0] );
-
-  real64 X5_X0[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X[5] );
-  LvArray::tensorOps::subtract< 3 >( X5_X0, X[0] );
-
-  real64 X7_X4[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X[7] );
-  LvArray::tensorOps::subtract< 3 >( X7_X4, X[4] );
-
-  real64 X7_X1plusX6_X0[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X7_X1 );
-  LvArray::tensorOps::add< 3 >( X7_X1plusX6_X0, X6_X0 );
-
-  real64 X7_X2plusX5_X0[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X7_X2 );
-  LvArray::tensorOps::add< 3 >( X7_X2plusX5_X0, X5_X0 );
-
-  real64 X7_X4plusX3_X0[ 3 ] = LVARRAY_TENSOROPS_INIT_LOCAL_3( X7_X4 );
-  LvArray::tensorOps::add< 3 >( X7_X4plusX3_X0, X3_X0 );
-
-  real64 X7_X2crossX3_X0[3];
-  LvArray::tensorOps::crossProduct( X7_X2crossX3_X0, X7_X2, X3_X0 );
-
-  real64 X7_X2plusX5_X0crossX7_X4[3];
-  LvArray::tensorOps::crossProduct( X7_X2plusX5_X0crossX7_X4, X7_X2plusX5_X0, X7_X4 );
-
-  real64 X5_X0crossX7_X4plusX3_X0[3];
-  LvArray::tensorOps::crossProduct( X5_X0crossX7_X4plusX3_X0, X5_X0, X7_X4plusX3_X0 );
-
-  return 1.0/12.0 * ( LvArray::tensorOps::AiBi< 3 >( X7_X1plusX6_X0, X7_X2crossX3_X0 ) +
-                      LvArray::tensorOps::AiBi< 3 >( X6_X0, X7_X2plusX5_X0crossX7_X4 ) +
-                      LvArray::tensorOps::AiBi< 3 >( X7_X1, X5_X0crossX7_X4plusX3_X0 ) );
+  constexpr integer numQuadraturePoints = 8;
+  real64 result =  finiteElement::H1_Hexahedron_Lagrange1_GaussLegendre2::transformedQuadratureWeight( 0, X );
+  for( localIndex q=1; q<numQuadraturePoints; ++q )
+  {
+    result = result + finiteElement::H1_Hexahedron_Lagrange1_GaussLegendre2::transformedQuadratureWeight( q, X );
+  }
```

Comment:
```suggestion
  real64 result =  0;
  for( localIndex q=0; q<numQuadraturePoints; ++q )
  {
    result = result + finiteElement::H1_Hexahedron_Lagrange1_GaussLegendre2::transformedQuadratureWeight( q, X );
  }
```
Won't the compiler be able to optimize the dummy `0` initialization of this loop?

Reply:
Done.